<launch>
  <node name="cam_left" pkg="usb_cam" type="usb_cam_node" output="screen" ns="tracking">
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="1920" />
    <param name="image_height" value="1080" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="tracking_cam_left" />
    <param name="camera_name" value="tracking_cam_left" />
    <param name="camera_info_url" value="file://$(find manual_control)/launch/tracking_cam_left.yaml"/>
    <param name="io_method" value="mmap"/>
    <param name="focus" value="0"/>
    <param name="sharpness" value="130"/>
    <param name="autoexposure" value="false"/>
    <param name="exposure" value="100"/>
    <param name="gain" value="50"/>
  </node>

  <node name="cam_right" pkg="usb_cam" type="usb_cam_node" output="screen" ns="tracking">
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="1920" />
    <param name="image_height" value="1080" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="tracking_cam_right" />
    <param name="camera_name" value="tracking_cam_right" />
    <param name="camera_info_url" value="file://$(find manual_control)/launch/tracking_cam_right.yaml"/>
    <param name="io_method" value="mmap"/>
    <param name="focus" value="0"/>
    <param name="sharpness" value="130"/>
    <param name="autoexposure" value="false"/>
    <param name="exposure" value="100"/>
    <param name="gain" value="50"/>
  </node>

  <node name="cam_middle" pkg="usb_cam" type="usb_cam_node" output="screen" ns="tracking">
    <param name="video_device" value="/dev/video2" />
    <param name="image_width" value="1920" />
    <param name="image_height" value="1080" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="tracking_cam_middle" />
    <param name="camera_name" value="tracking_cam_middle" />
    <param name="camera_info_url" value="file://$(find manual_control)/launch/tracking_cam_middle.yaml"/>
    <param name="io_method" value="mmap"/>
    <param name="focus" value="0"/>
    <param name="sharpness" value="130"/>
    <param name="autoexposure" value="false"/>
    <param name="exposure" value="100"/>
    <param name="gain" value="50"/>
  </node>
</launch>
